tests/test_downloader.py

```
@@ -47,6 +47,35 @@ def _write(dest, data=b"data"):
+def mock_github_commit_timestamp(commit_timestamps):
+    """
+    Create a mock function for GitHub API commit timestamp responses.
+
+    Parameters:
+        commit_timestamps (dict): Mapping of commit hash to ISO 8601 timestamp string
+
+    Returns:
+        function: Mock function that can be used with mock_get.side_effect
+    """
+
+    def mock_get_response(url, **kwargs):
```

Ruff (0.14.1)

Unused function argument: `kwargs`

(ARG001)